tools/gulptasks/scripts-webpack.js

```diff
@@ -34,6 +167,15 @@ async function scriptsWebpack() {
         configs = {
             Grid: 'grid.webpack.mjs'
         };
+    } else if (argv.dashboards) {
+        configs = {
+            Dashboards: 'dashboards.webpack.mjs'
+        };
+    } else if (argv.product === 'Dashboards') {
+        configs = {
+            Grid: 'grid.webpack.mjs',
```

Comment:
We probably don't need to build Grid along with Dashboards, they should be now completely separated, independent products.